src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/VSActionsTelemetryEvent.cs

```diff
@@ -4,11 +4,16 @@
 using System;
 using NuGet.Common;
 using NuGet.PackageManagement;
+using NuGet.VisualStudio.Internal.Contracts;
 
 namespace NuGet.VisualStudio
 {
     public class VSActionsTelemetryEvent : ActionsTelemetryEvent
     {
+        public bool IsSolutionLevel { set => this["IsSolutionLevel"] = value; }
```

Comment:
I will use `IsSolutionLevel` since this property name is already used in [PackageManagerUIRefreshEvent](https://github.com/NuGet/NuGet.Client/blob/bc4b41f3ca294ccc8aedc8af5159c215943e1ed2/src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/PackageManagerUIRefreshEvent.cs#L24) and both properties have the same meaning.